// Copyright 2025 The Meridian Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

edition = "2023";

package mmm.v1.common;

import "google/type/date.proto";

option features.field_presence = IMPLICIT;
option java_multiple_files = true;

message DateInterval {
  // The start date of the interval. Inclusive. Required.
  google.type.Date start_date = 1;

  // The end date of the interval. Exclusive. Required.
  google.type.Date end_date = 2;

  // A tag to identify the date interval. Optional.
  string tag = 3;
}
